spinnaker-monitoring-daemon/spinnaker-monitoring/metric_filter.py

```diff
@@ -148,10 +149,13 @@ class MetricFilter(object):
                             and their MeterSpec specifying tags.
   """
 
-  def __init__(self, filter_spec):
+  def __init__(self, filter_spec, **kwargs):
     """Initialize a new metric filter using the given specification."""
     self.__init_tags(filter_spec)
     self.__init_meters(filter_spec)
+    self.__filter_name = kwargs.pop('name', '')
```

Comment:
Looks like the logging looks strange if we default to an empty string here, maybe default to a placeholder string instead e.g. `DEFAULT_FILTER` or something.